test/App.spec.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import { configure, mount } from 'enzyme';
+import renderer from 'react-test-renderer';
+
+import App, { Counter } from '../client/src/App';
+
+configure({ adapter: new Adapter() });
+
+describe('App', () => {
+  test('snapshot renders', () => {
+    const component = renderer.create(<App />);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  // enzyme testing
+  it('renders the inner Counter', () => {
+    const wrapper = mount(<App />);
```

Comment:
this is interesting! guess this one in particular will be easy to replicate for our own components